### PR TITLE
Unit test blitz

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

This PR fixes the "importer" unit tests under blitz

# Testing this PR

Check that the units are run and are green

# Related reading
Noticed while working on https://trello.com/c/9nZfJ6Pi/132-import-static-cleanup

### CODE REVIEW

File: components/blitz/test/ome/formats/importer/OMEROWrapperTest.java [hunk 1]
```diff
@@ -102,57 +103,55 @@ public void testPNG() throws Exception {
...
-        }
+        for (int i = 0; i < 10; i++) {
+            try (IFormatReader r = new MinMaxCalculator();
+                 Memoizer m = new Memoizer(0L);
+                 Memoizer m1 = new Memoizer(r, 0L)) {
+                m.setId(fake.getAbsolutePath());
+                m.close();
+                Assert.assertTrue(m.isSavedToMemo());
+                m1.setId(fake.getAbsolutePath());
+                Assert.assertFalse(m1.isLoadedFromMemo());
```
Comment (mtbc) [inline components/blitz/test/ome/formats/importer/OMEROWrapperTest.java]: does `m1` need closing?

File: components/blitz/test/unit.testng.xml [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
       <run>
         <exclude name="broken"/>
         <exclude name="ignore"/>
         <exclude name="integration"/>
-      </run>        
+      </run>
     </groups>
     <packages>
       <package name="ome.formats.*">
-      <exclude name="ome.formats.importer"/>
```
Comment (mtbc) [inline components/blitz/test/unit.testng.xml:15]: :+1:

File: components/blitz/test/ome/formats/importer/OMEROWrapperTest.java (hunk 1 above)
Comment (jburel) [reply components/blitz/test/ome/formats/importer/OMEROWrapperTest.java]: it will be closed

**Comment by mtbc** [comment]
Locally with,
```
./build.py -f components/blitz/build.xml test
```
I then get,
```
===============================================
blitz.unit
Total tests run: 47, Failures: 1, Skips: 0
===============================================
```
and `components/blitz/target/reports/xml/ome.formats.importer.OMEROWrapperTest_results.xml` reports,
```xml
      <failure type="org.testng.internal.thread.ThreadTimeoutException"
                  message="Method org.testng.internal.TestNGMethod.testMismatchedWrappers() didn&apos;t finish within the time-out 1000"
              >
<![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.testMismatchedWrappers() didn't finish within the time-out 1000
  java.lang.Object.hashCode(Native Method)
  com.esotericsoftware.kryo.util.ObjectMap.push(ObjectMap.java:218)
  com.esotericsoftware.kryo.util.ObjectMap.putResize(ObjectMap.java:182)
  com.esotericsoftware.kryo.util.ObjectMap.resize(ObjectMap.java:488)
  com.esotericsoftware.kryo.util.ObjectMap.put_internal(ObjectMap.java:137)
  com.esotericsoftware.kryo.util.ObjectMap.put(ObjectMap.java:78)
  com.esotericsoftware.kryo.util.DefaultClassResolver.register(DefaultClassResolver.java:50)
  com.esotericsoftware.kryo.util.DefaultClassResolver.registerImplicit(DefaultClassResolver.java:56)
  com.esotericsoftware.kryo.Kryo.getRegistration(Kryo.java:461)
  com.esotericsoftware.kryo.Kryo.getSerializer(Kryo.java:476)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:109)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:378)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:289)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:378)
  com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ObjectArraySerializer.read(DefaultArraySerializers.java:289)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
  com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106)
  com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:528)
  com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:657)
  loci.formats.Memoizer$KryoDeser.loadReader(Memoizer.java:163)
  loci.formats.Memoizer.loadMemo(Memoizer.java:866)
  loci.formats.Memoizer.setId(Memoizer.java:653)
  ome.formats.importer.OMEROWrapperTest.testMismatchedWrappers(OMEROWrapperTest.java:131)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
  org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
  java.util.concurrent.FutureTask.run(FutureTask.java:262)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>
```
With,
```
./components/tools/travis-build java-test
```
I thus get,
```
BUILD FAILED
```

**Comment by jburel** [comment]
timeout!

File: components/blitz/test/ome/formats/importer/OMEROWrapperTest.java [hunk 3]
```diff
@@ -102,58 +103,57 @@ public void testPNG() throws Exception {
...
-            m = new Memoizer(new ChannelSeparator());
-            m.setId(png.getAbsolutePath());
+            m.setId(fake.getAbsolutePath());
             Assert.assertTrue(m.isLoadedFromMemo());
-        } finally {
             m.close();
         }
     }
 
     @Test
```
Comment (mtbc) [inline components/blitz/test/ome/formats/importer/OMEROWrapperTest.java]: would,
```java
@Test(invocationCount = 10)
```
be valid and less likely to time out?

Comment (jburel) [reply components/blitz/test/ome/formats/importer/OMEROWrapperTest.java]: That's probably a good option. 
Pushing a fix

**Comment by jburel** [comment]
@mtbc tests are green
ok to merge?

**Comment by mtbc** [comment]
```
===============================================
blitz.unit
Total tests run: 56, Failures: 0, Skips: 0
===============================================
```

**Review by mtbc — APPROVED** [review]

**Pull request merged**